cmd/argocd/commands/admin/settings_rbac.go

```diff
@@ -191,24 +193,45 @@ argocd admin settings rbac can someuser create application 'default/app' --defau
 }
 
 // NewRBACValidateCommand returns a new rbac validate command
-func NewRBACValidateCommand() *cobra.Command {
+func NewRBACValidateCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 	var (
 		policyFile string
 	)
 
 	var command = &cobra.Command{
-		Use:   "validate --policy-file=POLICYFILE",
+		Use:   "validate --policy-file=POLICYFILE [--namespace=NAMESPACE]",
 		Short: "Validate RBAC policy",
 		Long: `
 Validates an RBAC policy for being syntactically correct. The policy must be
 a local file, and in either CSV or K8s ConfigMap format.
 `,
 		Run: func(c *cobra.Command, args []string) {
-			if policyFile == "" {
+			if len(args) > 0 {
 				c.HelpFunc()(c, args)
-				log.Fatalf("Please specify policy to validate using --policy-file")
+				os.Exit(1)
```

Comment:
nit: Should we log a message here before exiting?

Reply:
@ishitasequeira I've done the needful in the commit below can you please review it?

Reply:
@ishitasequeira I've done the needful in the commit below, can you please review it